package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0.net",
-  "version": "0.0.1",
+  "version": "7.16.1",
```

Comment:
This doesnt matter, but I believe it makes sense to have this set correctly anyway. But it's not an npm package so shouldn't matter.